shared/js/otSdkDriver.js

```diff
@@ -502,6 +502,8 @@ loop.OTSdkDriver = (function() {
           break;
         case "Session.networkDisconnected":
         case "Session.forceDisconnected":
+        case "Session.subscribeCompleted":
+        case "Session.screen.subscribeCompleted":
```

Comment:
We do log metrics - these aren't ones that bump the counts but are also valid ones. The default if not listed is to warn and exit the function.